backend/internal/pkg/routes/session.go

```diff
@@ -31,6 +31,15 @@ type SessionDataGetter interface {
 	Get(ctx context.Context, key string) any
 }
 
+type SessionDataPutter interface {
```

Comment:
Thanks for adding this.  Hritik and I were confused on Friday's class why there was only a Getter interface and no Putter.